tfrecorder/check.py

```diff
@@ -90,8 +85,11 @@ def check_tfrecords(
     writer = csv.writer(f)
 
     # Write CSV header
-    header = [k for k in constants.RAW_FEATURE_SPEC.keys() if k != 'image']
-    writer.writerow(header)
+    for data in dataset.take(1).as_numpy_iterator():
+      # .as_numpy_iterator() converts from Tuple of Tensors to a dict.
+      # list() yields the keys of that dict.
+      header = [k for k in list(data) if k != 'image']
```

Comment:
Maybe make the if statement `if 'image' not in k` to generalize a bit. This assumes that image URI fields would have the 'image' substring in them, so we should probably document that as well.

Reply:
Okay, makes sense.  I think in the future it might be good to get the columns schema type, but maybe we can defer that for now and just require the usage of image in the name.

Reply:
I actually had to revert this, otherwise it will remove the other image features with image in the substring like image_height

Reply:
Got it. We need to distinguish the image bytes from the image metadata. Maybe we can just add a todo for now.